airflow/executors/kubernetes_executor.py

```diff
@@ -122,11 +132,13 @@ def as_dict(self):
         }
 
 
-class KubeConfig:
+class KubeConfig:  # pylint: disable=too-many-instance-attributes
+    """Configuration for Kubernetes"""
     core_section = 'core'
     kubernetes_section = 'kubernetes'
 
     def __init__(self):
+        """Creates object"""
```

Comment:
```suggestion
```
It is not necessary. We should keep the documentation in the class description

Reply:
Done.